packages/xo-web/src/xo-app/dashboard/health/unhealthyVdis.js

```diff
@@ -80,6 +115,26 @@ const UnhealthyVdis = decorate([
           </CardBlock>
         </Card>
       </Col>
+
+      <Col>
+        <Card>
+          <CardHeader>
+            <Icon icon='disk' /> {_('vdisWithInvalidVhdParent')}
+          </CardHeader>
+          <CardBlock>
+            <Row>
+              <Col>
+                <SortedTable
+                  data-chainsLengthBySr={chainsLengthBySr}
+                  collection={srIdsWithUnknownVdisParent}
+                  columns={COLUMNS_VDI_UNKNOWN_PARENT}
+                  stateUrlParam='s_vdis_invalid_parent'
+                />
+              </Col>
+            </Row>
+          </CardBlock>
+        </Card>
+      </Col>
```

Comment:
IMHO,  we don't need to add a new table, we can add a column in the existing table that contains the orphaned VDIs.